include/exadg/postprocessor/pressure_difference_calculation.cpp

```diff
@@ -25,10 +25,37 @@
 // ExaDG
 #include <exadg/postprocessor/pressure_difference_calculation.h>
 #include <exadg/utilities/create_directories.h>
+#include <exadg/utilities/print_functions.h>
 #include <exadg/vector_tools/point_value.h>
 
 namespace ExaDG
 {
+template<int dim>
+void
+free_print(dealii::ConditionalOStream &        pcout,
```

Comment:
I would prefer to realize this as a member function.

Reply:
[e482ac8](https://github.com/exadg/exadg/pull/262/commits/e482ac8101da49a4f0c27e76d4da9a75f683a836)